superdesk/commands/index_from_mongo.py

```diff
@@ -36,23 +37,24 @@ class IndexFromMongo(superdesk.Command):
         superdesk.Option("--from", "-f", dest="collection_name"),
         superdesk.Option("--all", action="store_true", dest="all_collections"),
         superdesk.Option("--page-size", "-p"),
+        superdesk.Option("--last-id"),
     ]
     default_page_size = 500
 
-    def run(self, collection_name, all_collections, page_size):
+    def run(self, collection_name, all_collections, page_size, last_id):
         if not collection_name and not all_collections:
             raise SystemExit("Specify --all to index from all collections")
         elif all_collections:
             app.data.init_elastic(app)
             resources = app.data.get_elastic_resources()
             for resource in resources:
-                self.copy_resource(resource, page_size)
+                self.copy_resource(resource, page_size, last_id)
```

Comment:
imo it only makes sense to use the last id with single resource, so if it's set with all collections it might stop